packages/gatsby/src/utils/webpack/corejs-resolver.ts

```diff
@@ -0,0 +1,103 @@
+import Resolver from "enhanced-resolve/lib/Resolver"
```

Comment:
Since this is only being used for a type, I think you should be able to import it from the parent lib because of this line: https://github.com/webpack/enhanced-resolve/blob/master/types.d.ts#L393


`import { Resolver } from "enhanced-resolve"` should work

Reply:
https://github.com/webpack/enhanced-resolve/tree/v4.2.0 doesn't has types yet :'(